src/index.html

```diff
@@ -4,9 +4,41 @@
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
- <title>Document</title>
+ <title>Basic Page Structure</title>
 </head>
 <body>
- 
+ <div class="wrapper">
+  <header class="header">
+   <h1>Leaderboard</h1>
+  </header>
+  <main class="main-section">
+   <div class="recent-scores">
+    <div class="recent-head">
+     <h2>Recent scores</h2>
+     <button type="submit" class="refresh">Refresh</button>
+    </div>
+    <ul class="score-list">
+     <li class="score-item"><p>Name:<span>100</span></p></li>
+     <li class="score-item"><p>Name:<span>20</span></p></li>
+     <li class="score-item"><p>Name:<span>50</span></p></li>
+     <li class="score-item"><p>Name:<span>78</span></p></li>
+     <li class="score-item"><p>Name:<span>125</span></p></li>
+     <li class="score-item"><p>Name:<span>77</span></p></li>
+     <li class="score-item"><p>Name:<span>42</span></p></li>
+    </ul>
+   </div>
+   <form class="add-scores">
+    <h2>Add your score</h2>
+    <div class="name">
+     <input type="text" placeholder="Your name" aria-label="name" class="name-input">
+    </div>
+    <div class="score">
+     <input type="text" placeholder="Your score" aria-label="name" class="score-input">
+    </div>
+    <button type="submit" class="submit-btn">Submit</button>
+   </form>
+  </main>
+  <footer class="footer-section"></footer>
+ </div>
```

Comment:
- [ ] index.html main contain only root element. In other words, the code you made is excellent, now let's convert it to js code inside index.js. 🐶 
- hint: Try to use `<div id="root"></div>` on index.html and use the index.js to create the project with js code.
- Don't forget to use ES6 best practices.